prime-router/metadata/schemas/AZ/az-covid-19-csv.schema

```diff
@@ -48,10 +48,10 @@ elements:
   - name: standard.patient_gender
     csvField: Patient_Sex
 
-  - name: standard.patient_race
+  - name: standard.patient_race#text
```

Comment:
#text is the magic that tells code to use the "display" value instead of the "code" value.    These changes were to meet AZ requests.